src/Components/GameBoard.js

```diff
@@ -679,7 +679,31 @@ class GameBoard extends Component {
     )
   }
 
+  afterRenderEdge = (id, element, viewEdge, edgeContainer,isEdgeSelected) => {
+
+    //afterRenderEdge?: (id: string, element: any, edge: IEdge, edgeContainer: any, isEdgeSelected: boolean) => void;
+    console.log("Start afterRenderEdge");
+    console.log("id= ",id);
+    console.log("element= ",element);
+    console.log("viewEdge= ",viewEdge);
+    console.log("edgeContainer= ",edgeContainer);
+    console.log("isEdgeSelected= ",isEdgeSelected);
+    console.log("end afterRenderEdge");
```

Comment:
console.logs should be removed or commented

Reply:
Console log's are still present @ryancb4